app/workers/migrate_data_to_work_field_worker.rb

```diff
@@ -0,0 +1,13 @@
+# NOTE: we can remove this once the DUS using it finished
```

Comment:
This is true for Forem instances that are always kept up-to-date with the latest releases, but for instances that are a few months behind this may not be true. We should come up with a deprecation/deletion strategy for stuff like this so some Forems don't end up losing data.

Reply:
What I meant by "finished" was "run across the fleet". I'll clarify the comment a bit.